public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryEditorSelector.tsx

```diff
@@ -51,9 +51,14 @@ export const PromQueryEditorSelector = React.memo<Props>((props) => {
   const [dataIsStale, setDataIsStale] = useState(false);
   const { flag: explain, setFlag: setExplain } = useFlag(promQueryEditorExplainKey);
 
+  const llmSrv = getLLMSrv();
+
   const query = getQueryWithDefaults(props.query, app, defaultEditor);
   // This should be filled in from the defaults by now.
-  const editorMode = query.editorMode!;
+  let editorMode = query.editorMode!;
+  if (llmSrv === undefined && editorMode === QueryEditorMode.Natural_language) {
+    editorMode = QueryEditorMode.Builder;
+  }
```

Comment:
It would be much more explicit if we did something like `!llmSrv.isEnabled` in this case compared to `llmSrv === undefined`.